test/index.coffee

```diff
@@ -77,6 +78,192 @@ describe 'Coverting HTML to MHT', ->
     expect(utils.getMHTdocument(htmlSource)).to.match
     '<body style=3D"width: 100%">This 3D= 0</body>'
 
+describe 'Inlining CSS', ->
```

Comment:
I think it makes more sense to move these tests to a separate `css_inliner_test` file.